src/prebid.js

```diff
@@ -37,6 +37,7 @@ var eventValidators = {
 
 $$PREBID_GLOBAL$$._bidsRequested = [];
 $$PREBID_GLOBAL$$._bidsReceived = [];
+$$PREBID_GLOBAL$$._adUnitCodes = [];
```

Comment:
The issue here is that you can't know what adUnits are currently used in the current auction... But maybe that's because I've got rid of `markComplete`.